META.d/_summary.yml

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+schema: 1.1
+partition: tf-ecosystem
```

Comment:
@deniseyu @xiehan The partition is loosely defined in the schema. Is there a better choice here?

Reply:
No idea. I personally have been against implementing Heimdall cataloguing for open source repos, the only thing I've implemented it for are private repos tangentially related to TFC so I used "tfc" for those.

Is there a reason why we're doing this for these repos now?